repository/src/main/java/org/apache/atlas/repository/store/graph/v2/preprocessor/sql/QueryFolderPreProcessor.java

```diff
@@ -80,12 +91,211 @@ private void processCreate(AtlasStruct entity) throws AtlasBaseException {
         entity.setAttribute(QUALIFIED_NAME, createQualifiedName(collectionQualifiedName));
     }
 
-    private void processUpdate(AtlasStruct entity, AtlasVertex vertex) {
-        String vertexQnName = vertex.getProperty(QUALIFIED_NAME, String.class);
+    private void processUpdate(AtlasEntity entity, AtlasVertex vertex, EntityMutationContext context) throws AtlasBaseException {
```

Comment:
Code duplication in processUpdate and processUpdateQueryCollection

Reply:
Added one single function in PreProcessorUtils to address this